pom.xml

```diff
@@ -1139,7 +1139,7 @@
                 <plugin>
                     <groupId>io.airlift.maven.plugins</groupId>
                     <artifactId>sphinx-maven-plugin</artifactId>
-                    <version>2.0</version>
+                    <version>2.1</version>
```

Comment:
> `Update Sphinx plugin to 2.1`

Any particular reason or "just update"?

Reply:
It both updates the underlying Sphinx library and fixes illegal access warnings on Java 9.